.eslintrc.json

```diff
@@ -1,33 +1,17 @@
 {
-  "env": {
-    "node": true,
-    "es6": true
-  },
-  "extends": "eslint:recommended",
+  "parser": "@typescript-eslint/parser",
+  "extends": ["plugin:@typescript-eslint/recommended"],
+  "plugins": ["@typescript-eslint"],
   "rules": {
-    "indent": [
+    "@typescript-eslint/indent": [
       "error",
       2
     ],
-    "quotes": [
-      "error",
-      "single"
-    ],
-    "semi": [
-      "error",
-      "always"
-    ],
-    "no-console": 0
-  },
-  "parserOptions": {
-    "ecmaVersion": 2018,
-    "sourceType": "module",
-    "ecmaFeatures": {
-      "arrowFunctions": true,
-      "classes": true,
-      "impliedStrict": true,
-      "modules": true,
-      "spread": true
-    }
+    "@typescript-eslint/no-explicit-any": 0,
```

Comment:
Ignoring these rules for now, will be fixing the issues (unless ignoring them makes sense) in a future PR